scripts/ts-utils/index.ts

```diff
@@ -0,0 +1,16 @@
+const { ApiPromise, WsProvider } = require("@polkadot/api");
+
+//fetchApi is used to fetch data from the chain
+const wsFetchProvider = new WsProvider("wss://tee-staging.litentry.io:443");
+
+//syncApi is used to send transactions to the chain
+const wsSyncProvider = new WsProvider("ws://localhost:9944");
+
```

Comment:
I think these two urls should be input parameter for this module.

Reply:
`wsFetchProvider`is the endpoint get vcRegistry, `wsSyncProvider` is the endpoint we need to synchronize,encode call need to specify an endpoint,otherwise can't decode,so we should define two endpoint

Reply:
understand. What about to rename `fetch` & `sync` to be more specific names?